docs/program/python-venv.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Prepare your Python Virtual Environment"
+linkTitle: "Prepare your Python Virtualenv for Viam"
+weight: 10
+type: "docs"
+description: "Prepare your Python Virtual Environment to program machines with the Python SDK."
+images: ["/services/icons/sdk.svg"]
+tags: ["client", "sdk", "application", "sdk", "fleet", "program", "python", "venv"]
+---
+
+To manage Python packages for your Viam application, it is recommended that you use a virtual environment, or `venv`.
+By using a `venv`, you can install Python packages like Viam's client SDK just within the virtual environment, and not globally, which could cause conflicts with other projects or cause issues within your system.
+
+Follow this guide to set up a fresh virtual environment on your working computer and install the Python SDK as a requirement for your Viam client application.
+
+## Setup your project
+
+First, create a directory for your project.
+For example, name your directory `viam-python`:
+
+```sh {class="command-line" data-prompt="$"}
+mkdir viam-python
+cd viam-python
+```
+
+## Create and Activate a Virtual Environment
+
+Now that you are in the project directory, create and activate a virtual environment for Python to run in.
+
+```bash
+python3 -m venv viam-env
+source viam-env/bin/activate
+```
+
+Now, `(viam-env)` prepends the commands in your terminal window to indicate the Python packages being used are from this particular environment.
+You can exit this environment by running `deactivate`.
+
+## Install Viam
+
+Inside the activated `viam-env` python environment, you can now install the Viam SDK:
+
+```bash
+pip3 install viam-sdk
+```
+
+This will install Viam and all required dependencies.
+
+Should you need to install your own requirements, be sure to do so in this virtual environment.
+If you have more requirements, you can [create a](https://openclassrooms.com/en/courses/6900846-set-up-a-python-environment/6990546-manage-virtual-environments-using-requirements-files) <file>requirements.txt</file> with all the packages you need and then install the requirements for your client application by running `pip3 install -r requirements.txt`.
```

Comment:
```suggestion
To make your required packages easier to install in the future, you can also [create a](https://openclassrooms.com/en/courses/6900846-set-up-a-python-environment/6990546-manage-virtual-environments-using-requirements-files) <file>requirements.txt</file> file with a list of all the packages you need and then install the requirements for your client application by running `pip3 install -r requirements.txt`.
```

At this point it's an alternative. the previous line kind of addressed what to do if they had more requirements. This is something for convenience. So we can explain that or just say "You can also..."

Reply:
Done